app/Judite/Models/Membership.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace App\Judite\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Membership extends Model
+{
+    /**
+     * The relations to eager load on every query.
+     *
+     * @var array
+     */
+    protected $with = ['student', 'group'];
+
+    /**
+     * Scope a query to filter memberships by owner.
+     *
+     * @param \Illuminate\Database\Eloquent\Builder $query
+     * @param \App\Judite\Models\Student            $student
+     *
+     * @return \Illuminate\Database\Eloquent\Builder
+     */
+    public function scopeOwnedBy($query, Student $student)
+    {
+        return $query->whereStudentId($student->id);
+    }
+
+    /**
+     * Scope a query to order memberships by students.
```

Comment:
Scope a query to order memberships by students number.

Reply:
👍